ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/hosts.yml

```diff
@@ -97,6 +97,8 @@ all:
               hosts:
                 UPLINK-MLAG-STRUCTURED-CONFIG-L2LEAF1A:
                 UPLINK-MLAG-STRUCTURED-CONFIG-L2LEAF1B:
+                TEST-MGMT-GATEWAY-IN-NODE-GROUP-NODES:
+                TEST-MGMT-GATEWAY-IN-NODE-GROUP1:
```

Comment:
move these new hosts to "CLEAN_UNIT_TESTS"

Reply:
Updated with a new test group